python/plugins/crew_ai/setup.py

```diff
@@ -24,7 +24,7 @@
     python_requires=">=3.9,<4",
     install_requires=[
         "composio_langchain>=0.5.50,<=0.5.52-rc.2",
-        "crewai>=0.51.0,<=0.75.0",
+        "crewai>=0.51.0",
```

Comment:
While removing the upper version limit provides more flexibility, it's recommended to add a comment here documenting:
1. The minimum version requirement rationale
2. That compatibility with newer versions has been tested
3. The latest tested version number

This will help future maintainers understand the version constraints and testing status.